WooCommerce/Classes/ViewRelated/Orders/Order Filters/FilterOrderListViewModel.swift

```diff
@@ -210,5 +232,30 @@ private extension FilterOrderListViewModel.OrderListFilter {
         static let rowTitleProduct = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowTitleProduct",
                                                        value: "Product",
                                                        comment: "Row title for filtering orders by Product.")
+        static let rowCustomer = NSLocalizedString("filterOrderListViewModel.OrderListFilter.rowCustomer",
+                                                   value: "Customer",
+                                                   comment: "Row title for filtering orders by customer.")
+    }
+}
+
+extension CustomerFilter: FilterType {
+    /// The user-facing description of the filter value.
+    var description: String {
+        let fullName = ((firstName ?? "") + " " + (lastName ?? "")).trimmingCharacters(in: .whitespacesAndNewlines)
+
+        if !fullName.isEmpty {
+            return fullName
+        } else if let email,
+                    !email.isEmpty {
+            return email
+        } else if let username,
+                  !username.isEmpty {
```

Comment:
Nit: we have `isNotEmpty` helper so we can use that for better readability.

Reply:
Nice, added in https://github.com/woocommerce/woocommerce-ios/pull/12100/commits/8cae88675837ce68d96b2bb427c1ced3e6819936